simple_ddl_parser/dialects/sql.py

```diff
@@ -221,7 +221,7 @@ def process_type(self, _type: Union[str, List], p_list: List, p: List) -> str:
             p[0] = {"property": {"distkey": True}}
             _type = _type.split("distkey")[0]
 
-        _type = _type.strip().replace('" . "', '"."')
+        _type = _type.strip().replace(" . ", ".")
```

Comment:
I can't think of a reason to want to keep the space around the dots if the strings are unquoted, so i simplified to this.  This then allows the PR i'm about to make to omymodels pass existing tests....

Reply:
again, looks like it was 'historical context'